infrastructure/src/main/java/org/corfudb/infrastructure/management/CompleteGraphAdvisor.java

```diff
@@ -126,7 +124,9 @@ public Optional<NodeRank> findDecisionMaker(ClusterState clusterState, Set<Strin
         ClusterGraph symmetric = ClusterGraph
                 .toClusterGraph(clusterState)
                 .toSymmetric();
-
-        return symmetric.getDecisionMaker(healthyNodes);
+        log.info("Symmetric: {}", symmetric);
```

Comment:
same here, please change to trace

Reply:
I will leave it as info for now. If we get a bug in prod, it's easier to debug a graph this way.